pugjs/types.go

```diff
@@ -328,6 +334,22 @@ func (a *Array) Pop() Object {
 	return last
 }
 
+// Shift removes an element from the beginning of the array and returns the element
+func (a *Array) Shift() Object {
+	if len(a.items) > 0 {
+		first := a.items[0]
+		a.items = a.items[1:len(a.items)]
```

Comment:
This could even be shorter, couldn't it:

```suggestion
		a.items = a.items[1:]
```

Reply:
Changed it. Unit tests are telling me that it is still working